src/Analyzers/Microsoft.Analyzers.Local/Microsoft.Analyzers.Local.csproj

```diff
@@ -8,9 +8,7 @@
 
   <PropertyGroup>
     <SkipLocalAnalyzers>true</SkipLocalAnalyzers>
-      <AnalyzerLanguage>cs</AnalyzerLanguage>
-    <AnalyzerRoslynVersion>4.0</AnalyzerRoslynVersion>
-    <DefineConstants>$(DefineConstants);ROSLYN_4_0_OR_GREATER</DefineConstants>
+    <AnalyzerLanguage>cs</AnalyzerLanguage>
```

Comment:
There's an established naming convention in the SDK to suffix assemblies with the language. E.g.,
* Microsoft.Analyzers.Local.dll implies a language agnostic analyzers,
* Microsoft.Analyzers.Local.CSharp.dll implies C#-specific analyzers.

It'd worth considering adapting to the same convention.